docs/source/index.rst

```diff
@@ -1,9 +1,58 @@
-.. include:: ../../README.rst
+.. image:: https://github.com/aesara-devs/aeppl/blob/main/docs/source/logo.png?raw=true
```

Comment:
Looks like this might not be rendering: see [here](https://aeppl--213.org.readthedocs.build/en/213/examples.html).